examples/pipelines/example_pipeline/components/example_component/README.md

```diff
@@ -0,0 +1,39 @@
+# example_component
+
+### Description
```

Comment:
Not sure if we need a new example pipeline with a single component if there's already an example pipeline added in #16 ?

Reply:
It guess we won't need it once we have the example HF pipeline, i'll remove it